### ISSUE TITLE
mine tower dialog disappears if dragging mouse into UI

### ISSUE THREAD

**Issue by heavyk** [MaFi-Games/Captain-of-Industry-issues#919]
**The issue**
the dialog should return to be visible again

**To Reproduce**
- pause the game
- select a mine tower
- click 'edit area'
- drag a box to any size, but let go of the mouse button over the 'paused' box or anywhere on the top bar
- neither the mine area is set or the mine tower dialog returns to view (it eats the mouse event)
 - same thing also happens if I right click or press esc (not sure if this is intentional or not)

0.4.13

**Comment by MaFi-Games** [comment]
Yes, releasing the mouse button on top of any UI is considered as aborted operation.

**Comment by heavyk** [comment]
I do agree for all aspects of the UI like the top and side bars, but I'm not sure about the little "paused" box.  I've learned to look now, but it still catches me. then, I have to go back to find the tower and reopen the dialog.

at least, if it's going to abort it, I would expect the mine tower dialog to show up again so I can try selecting the area again.

**Comment by Decavoid** [comment]
"Paused" box is not designed to respond to mouse clicks. It should have "Raycast target" Unity UI checkbox unchecked.
